### PR TITLE
Update multicluster app deployment using jenkinsfile

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

I will add some images later for the last part.

### CODE REVIEW

File: content/en/docs/devops-user-guide/examples/multi-cluster-project-example.md [hunk 1]
```diff
@@ -1,138 +1,155 @@
...
+
+   ![dockerhub-token-ok](/images/docs/devops-user-guide/examples/compile-and-deploy-a-go-multi-cluster-project/dockerhub-token-ok.jpg)
+
+4. Click **Copy and Close** and remember to save the access token.
+
+   ![dockerhub-token-copy](/images/docs/devops-user-guide/examples/compile-and-deploy-a-go-multi-cluster-project/dockerhub-token-copy.jpg)
+
+## Create Credentials
+
+You need to create credentials in KubeSphere for the access token created so that the pipeline can interact with Docker Hub for imaging pushing. Besides, you also create kubeconfig credentials for the access to the Kubernetes cluster.
```
Comment (yuswift) [inline content/en/docs/devops-user-guide/examples/multi-cluster-project-example.md:36]: `you also need to create ` may be better?
@Sherlock113

File: content/en/docs/devops-user-guide/examples/multi-cluster-project-example.md [hunk 2]
```diff
@@ -1,138 +1,155 @@
...
+       APP_NAME = 'devops-go-sample'
+       // ‘dockerhubid’ is the dockerhub credential id you created on ks console
+       DOCKERHUB_CREDENTIAL = credentials('dockerhubid')
+       // the kubeconfig credential id you created on ks console
+       KUBECONFIG_CREDENTIAL_ID = 'multi-cluster'
+       // mutli-cluster project name under your own workspace
+       MULTI_CLUSTER_PROJECT_NAME = 'devops-with-go'
+       // the member cluster name you want to deploy app on
+       // in this tutorial, you are assumed to deploy app on host and only one member cluster
+       // for more member clusters, please manifest/multi-cluster-deploy.yaml
```
Comment (yuswift) [inline content/en/docs/devops-user-guide/examples/multi-cluster-project-example.md:102]: please edit manifest/multi-cluster-deploy.yaml

**Review by yuswift — CHANGES_REQUESTED** [review]

**Comment by yuswift** [comment]
BTW, you can use `git rebase` to merge multiple commits into one finally, which can reduce the commits on master.

**Review by rayzhou2017 — APPROVED** [review]

**Pull request merged**